### ISSUE TITLE
No "latest" version sound, and go.mod error for v7.4.0 

### ISSUE THREAD

**Issue by Gattermeier** [mattermost/mmctl#594]
Following the README instructions:
`go install github.com/mattermost/mmctl@latest`
results in error: 
`go: github.com/mattermost/mmctl@latest: no matching versions for query "latest"`

Attempting to install the most recent release version: 
`go install github.com/mattermost/mmctl@v7.4.0`
results in error:
`go: github.com/mattermost/mmctl@v7.4.0: github.com/mattermost/mmctl@v7.4.0: invalid version: go.mod has post-v7 module path "github.com/mattermost/mmctl/v6" at revision v7.4.0`

**Comment by isacikgoz** [comment]
Hey @Gattermeier thanks for the feedback, sorry we had to update the readme after #428. Since we shifted module versions, now the latest wouldn't work as you already pointed out. 

Update it with the following PR #604

To get semantic versions, we are limited due to mismatch on the release tag and the module version. It's a long story but I built a tool to overcome this issue: https://github.com/isacikgoz/mmgoget

Let me know if this helps you.
